features/build_runner/fastlane_build_runner.rb

```diff
@@ -68,7 +70,7 @@ def run
                                                              .select { |value| value.to_s.include?("PATH") } # Far from ideal, but meanwhile...
                                                              .select { |value| !Fastlane::Actions.lane_context[value].nil? && !Fastlane::Actions.lane_context[value].empty? }
                                                              .map { |value| { type: value.to_s, path: Fastlane::Actions.lane_context[value] } }
-        return artifact_paths.concat(constants_with_path)
```

Comment:
Shouldn't "return from an ensure block"

Reply:
That's right, I think this piece will get re-written anyway once we decided on how to provide artifacts from fastlane